ui/DesignSystem/Component/AdditionalActionsDropdown.svelte

```diff
@@ -48,9 +49,11 @@
     right: 0;
     width: 240px;
     margin-top: 15px;
-    background-color: white;
-    box-shadow: 0px 4px 8px rgba(51, 51, 51, 0.08);
```

Comment:
Could you also use the variable here: https://github.com/radicle-dev/radicle-upstream/blob/master/ui/DesignSystem/Component/Sidebar.svelte#L90 while at it?